lib/puppet/ssl/ssl_provider.rb

```diff
@@ -148,6 +148,13 @@ def default_flags
   end
 
   def create_x509_store(roots, crls, revocation)
+    # TODO: This method is called in one Server test. (See:
```

Comment:
This is the test in question  https://github.com/puppetlabs/puppetserver/blob/6.5.0/test/integration/puppetlabs/services/master/master_service_test.clj#L664-L699 and I'm not sure yet whether the test is bad, Puppet's behavior is bad, or what.